src/content/reference/react/use.md

```diff
@@ -0,0 +1,769 @@
+---
+title: use
+---
+
+<Wip>
+This section is incomplete. See also the [draft use RFC](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md#usepromise).
+</Wip>
+
+[//]: # (TODO: replace the `<Note></Note>` below with `<Canary></Canary>` once https://github.com/reactjs/react.dev/pull/6173 is merged)
+<Note>
+The `use` hook is only supported when using [React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks). React Server Components are only available in React’s canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels). 
```

Comment:
Isn't `use` supported in canary and experimental without RSC?

Reply:
yes, you're right, this is a mistake! fixed